crates/ruff/src/rules/pycodestyle/snapshots/ruff__rules__pycodestyle__tests__W191_W19.py.snap

```diff
@@ -282,6 +282,16 @@ W19.py:133:1: W191 Indentation contains tabs
 137 | def test_keys(self):
     |
 
+W19.py:136:1: W191 Indentation contains tabs
```

Comment:
My understanding is that this was a false negative because `Indexer.strings` incorrectly suppressed this violation because it is on a line with a string. This now gets correctly reported because we test if the tab is inside of a string range (rather than on a line)

Reply:
I think you're right.